taxi_service/settings.py

```diff
@@ -108,6 +108,10 @@
 
 AUTH_USER_MODEL = "taxi.Driver"
 
+LOGIN_REDIRECT_URL = "/"
+
+LOGIN_URL = "/registration/login/"
```

Comment:
Why does the login URL have 'registration/login'? Is this a registration URL or a login URL?